mongodb_statedb/__init__.py

```diff
@@ -33,6 +33,20 @@ def get(self, key: str) -> Any:
         self._throw_error_if_key_doesnt_exist(key)
         return self._db.find_one({'_id': key})[self._VALUE_KEY]
 
+    def get_all_as_dict(self) -> dict:
```

Comment:
One related method that I would like to add would be the `keys`, like dictionaries have. I don't have the same hesitation with this as you still have to get the data, which is well managed by the MongoDB server.